.zuul.d/project.yaml

```diff
@@ -1,7 +1,14 @@
 - project:
     check:
       jobs:
+        - pyca-cryptography-build-wheel-arm64
+        - pyca-cryptography-build-wheel-x86_64
```

Comment:
We dont' usually run our wheel builder tasks in CI... though I guess there's no reason we couldn't. WDYT @reaperhulk ?

Reply:
I would say that this does ensure when it comes time to push a tag, you can be sure there's no surprises that you've broken something in the mean time.

Reply:
I don't have an objection to this happening provided we have the concurrency for it not to hurt our CI turnaround time.